dist/t/Gemfile.lock

```diff
@@ -47,4 +47,4 @@ DEPENDENCIES
   selenium-webdriver
 
 BUNDLED WITH
-   1.13.7
+   1.17.1
```

Comment:
Sure this wont break it?

Reply:
> Sure this wont break it?

Checking...

Reply:
It doesn't break it. Checked running smoke tests locally without problem.

Added a commit to change the repository to add, from openSUSE Leap 42.3 to openSUSE Leap 15.1.